Python/samples/quanser-qube/main.py

```diff
@@ -102,7 +102,13 @@ def halted(self) -> bool:
         """
         Should return True if the simulator cannot continue for some reason
         """
-        return False
+        sim_state = self.get_state()
+
+        # If arm hits rails of physical limit +/- 90 degrees
+        if abs(sim_state['theta']) > math.pi / 2:
```

Comment:
can simply do:

`abs(sim_state["theta"]) > math.pi / 2` here

Reply:
cool all for pythonic!